src/commands/deploy/genezio.ts

```diff
@@ -173,6 +173,11 @@ export async function genezioDeploy(options: GenezioDeployOptions) {
         const frontends = configuration.frontend;
 
         for (const [index, frontend] of frontends.entries()) {
+            // Skip if name option is provided and doesn't match the current frontend
+            if (options.name && frontend.name !== options.name) {
```

Comment:
`frontend.name` might be undefined because `name` is an optional field in the yaml.

A better UX would be if `options.name` is used, the user should be guided (via a warning log) to specify `frontend.name` for his frontends.

Otherwise - no frontend will be deployed because this condition will probably be false for every frontend.